package.json

```diff
@@ -56,8 +56,10 @@
     },
     "onlyBuiltDependencies": [
       "@swc/core",
+      "core-js",
```

Comment:
This should've been added to `ignoreBuiltDependencies`, but it doesn't really matter

Reply:
> This should've been added to `ignoreBuiltDependencies`, but it doesn't really matter

@lishaduck  Uh, pnpm was screaming at me so i did it... why would i not add it? I'm not totally familiar with this setting...

Reply:
onlyBuiltDependencies is for stuff that needs its postinstall to run, core-js just runs ads, so ignoredBuiltDependencies skips them (for security and perf)

Reply:
I moved it over in some PR.